tests/modules/programs/kubecolor/does-have-openshift.nix

```diff
@@ -0,0 +1,25 @@
+{ config, pkgs, ... }:
+
+{
+  programs.kubecolor = {
+    enable = true;
+    package = config.lib.test.mkStubPackage {
+      name = "kubecolor";
+      version = "0.4.0";
+    };
+    enableAlias = true;
+  };
+  programs.zsh = {
+    enable = true;
+    package = config.lib.test.mkStubPackage {
+      name = "zsh";
+      version = "5.9";
+    };
+  };
+  home.packages = [ pkgs.openshift ];
```

Comment:
@ajgon I added some tests for the conditional addition of the `oc` alias.

This is my first PR to nix-community. I'm not sure what the best way to test this would be. Is it bad to have `pkgs.openshift` in the test here, instead of `config.lib.test.mkStubPackage`?

Reply:
I think `mkStubPackage` will work better here, as it won't build whole derivation for openshift, just for testing. You can overload `pkgs` with overlay i.e.

```nix
  nixpkgs.overlays = [
    (self: super: rec {
      openshift = config.lib.test.mkStubPackage {
        name = "openshift";
        version = "4.16.0";
    };
  ];
```

and keep the rest of your test as is. This should use stub instead of a real package, thus avoiding building deriv.  'm not at my laptop right now, so can't guarantee if it works, but in general that's the idea you should follow :-).